redaxo/src/addons/structure/lib/structure_element.php

```diff
@@ -96,6 +96,24 @@ public function getValue($value)
         return null;
     }
 
+    /**
+     * Returns the value from this element or from the closest parent where the value is set.
+     *
+     * @return string|int|null
+     */
+    public function getClosestValue(string $value)
```

Comment:
Default als 2. param?

Reply:
Haben wir bei `getValue` in dieser Klasse auch nicht, würde ich daher hier glaube ich auch weglassen.
Man kann ja auch einfach `$cat->getClosestValue('foo') ?? 'default'` notieren.